plugins/org.jkiss.dbeaver.launcher/src/org/jkiss/dbeaver/launcher/DBeaverLauncher.java

```diff
@@ -2262,7 +2262,8 @@ private InputStream getStream(URL location) throws IOException {
     private void handleSplash(URL[] defaultPath) {
         // run without splash if we are initializing or nosplash
         // was specified (splashdown = true)
-        if (initialize || splashDown || bridge == null) {
+        boolean isRunningFromConsole = System.console() != null;
```

Comment:
Doesn't this mean the splash will be hidden in the development environment as well?